### PR TITLE
ASC: image_container

### PR DESCRIPTION
**Pull request by bmiles**

### CODE REVIEW

File: Drafts/image_container.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#### **Proposal**
+
+This proposal proposes a new instruction, and one to supersede the redacted `image_plate` instruction.
+
+```json
+{
+  "op": "image_container",
+  "objects": Container,
+  "view_point": ['above', 'side', 'below'],
```
Comment (rhysormond) [inline Drafts/image_container.md]: nit: preferred syntax is `Enum("above", "side", "below")`

File: Drafts/image_container.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "mode": 'visible_light'
```
Comment (rhysormond) [inline Drafts/image_container.md]: nit: prefer double quotes

**Comment by bmiles** [comment]
Addressed comments in https://github.com/autoprotocol/ascs/pull/8/commits/02981eda35803d22d8779de75994957ae21e30ce

**Comment by bmiles** [comment]
I guess I'll withdraw this due to #19

**Pull request closed without merging**